### DISCUSSION TITLE
nostr-relay docker image for ARM64 architecture

### DISCUSSION THREAD

**Discussion by lfnt3** [vishalxl/nostr_console#72] [Q&A] [answered]
I would like to host a nostr-relay on my Raspberry Pi4 using docker.
Unfortunately the [scsibug/nostr-rs-relay docker image](https://hub.docker.com/r/scsibug/nostr-rs-relay) is only available for AMD64, but for RP4 I need ARM64.

Is there an ARM64 image available somewhere? or would it be easy to build one?

Thanks!

**Comment by vishalxl** [comment, marked as answer]
Should ask in the nostr telegram channel, or to devs of that specific (or other) relays. They'd know best...
